index.html

```diff
@@ -1 +1,3 @@
-<p>메인입니다...</p>
\ No newline at end of file
+<p>메인입니다...</p>
+<p>둘째줄이네요...</p>
+<p>LGOM</p>
```

Comment:
단어가 잘못된 거 같습니다 확인해주세요 :)
```suggestion
<p>LGTM</p>
```

Reply:
예알겠씁니다 ^^7

Reply:
2f07a3b 수정했어요